resource_booking/models/resource_booking.py

```diff
@@ -231,31 +230,53 @@ def _compute_state(self):
             one.state = "scheduled" if one.meeting_id else "pending"
         to_check._check_scheduling()
 
-    @api.depends("meeting_id.start", "meeting_id.stop")
-    def _compute_dates(self):
+    @api.depends("meeting_id.start")
+    def _compute_start(self):
+        """Get start date from related meeting, if available."""
         for one in self:
-            # You're creating a new record; calendar view sends proper context
-            # defaults that at this point are lost; restoring them
+            if one.id:
+                one.start = one.meeting_id.start
+
+    @api.depends("meeting_id.duration", "type_id")
+    def _compute_duration(self):
+        """Compute duration for each booking."""
+        for one in self:
```

Comment:
I don't like too much this notation of `one` and prefer `record`, that is more standard.